dask/distributed/ipython_utils.py

```diff
@@ -15,19 +15,19 @@ def dask_client_from_ipclient(ipclient):
     def start_scheduler():
         from dask.distributed import Scheduler
         scheduler = Scheduler()
-        return scheduler.to_clients, scheduler.to_workers
+        return scheduler.address_to_clients, scheduler.address_to_workers
 
-    def start_worker(scheduler_port):
+    def start_worker(scheduler_address):
         from dask.distributed import Worker
-        worker = Worker(scheduler_port)
+        worker = Worker(scheduler_address)
 
-    # start sched
-    scheduler_target = ipclient[0]
+    # start scheduler
+    scheduler_target = ipclient[ipclient.ids[0]]
```

Comment:
Alternately, for symmetry with the worker slice, this could be `ipclient[:1]`, which would have targets as a list containing only the first engine. That would require a `[0]` on the apply_sync below.